lib/src/solver/version_solver.dart

```diff
@@ -431,7 +431,9 @@ class VersionSolver {
       List<PackageId> packages) async {
```

Comment:
Should we specify that `_getAvailableVersions` never includes retracted versions, instead? Where is the result used?

Reply:
I agree, done.